src/util/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# =============================
+# Define targets
+# =============================
+add_executable(siemens_icvutil icvutil.c)
+add_executable(siemens::icvutil ALIAS siemens_icvutil)
+
+set_target_properties(siemens_icvutil PROPERTIES
+    EXPORT_NAME icvutil
+    OUTPUT_NAME icvutil
+)
+
+target_compile_features(siemens_icvutil PRIVATE c_std_90)
+set_target_properties(siemens_icvutil PROPERTIES C_EXTENSIONS OFF)
+target_compile_options(siemens_icvutil PRIVATE
+    -Woverflow -Wswitch -Wmissing-prototypes -Wstrict-prototypes
+    -Wtype-limits -Wundef -Wconversion -Wno-shadow -Wno-conversion
+    -Wno-sign-conversion -Wno-unused-parameter -Wno-sign-compare # TODO enable -Wsign-conversion -Wsign-compare -Wunused-parameter
+    -Wformat -Wformat-security -Wno-declaration-after-statement -Wno-vla # -Wpointer-arith -pedantic -DPEDANTIC # -Werror
+    -pedantic -Werror
+)
+
```

Comment:
It is not good to define this twcie.
Can't this be inherited here and in `src/libsecutils` by defining it once in `src/CMakeLists.txt`?